src/legacy/core_plugins/kibana/public/dashboard/panel/panel_utils.ts

```diff
@@ -82,38 +109,40 @@ export class PanelUtils {
     return panel;
   }
 
-  static parseVersion(version = '6.0.0') {
+  public static parseVersion(version = '6.0.0') {
```

Comment:
could add type annotation for return value as well?

Reply:
Perfect! Adding an interface SemanticVersion to return the object with major and minor keys.

Reply:
fc7f3db